HousingSearchApi/V1/Domain/Person.cs

```diff
@@ -0,0 +1,103 @@
+using System.Collections.Generic;
+using HousingSearchApi.V1.Domain.ES;
+
+namespace HousingSearchApi.V1.Domain
+{
+    public class Person
+    {
+        public static Person Create(QueryablePerson person)
+        {
+            return new Person
+            {
+                Id = person.Id,
+                Title = person.Title,
+                Firstname = person.Firstname,
+                MiddleName = person.MiddleName,
+                Surname = person.Surname,
+                PreferredFirstname = person.PreferredFirstname,
+                PreferredSurname = person.PreferredSurname,
+                Ethinicity = person.Ethinicity,
+                Nationality = person.Nationality,
+                PlaceOfBirth = person.PlaceOfBirth,
+                DateOfBirth = person.DateOfBirth,
+                Gender = person.Gender,
+                Identification = Create(person.Identification ?? new List<ES.Identification>()),
+                PersonTypes = person.PersonTypes,
+                IsPersonCautionaryAlert = person.IsPersonCautionaryAlert,
+                IsTenureCautionaryAlert = person.IsTenureCautionaryAlert,
+                Tenures = Create(person.Tenures ?? new List<ES.Tenures>())
+            };
+        }
```

Comment:
Should this not be in the factories folder (or at least seperate from this file)

Reply:
Not necessarily, it's just another form of constructing an object from within itself :)

Reply:
But then it has a dependency on `QueryablePerson` - it might make sense to flip the dependency and have it on the gateway class to _create_ a Domain entity

Reply:
Hm - I would have thought that QueryablePerson sits (at least) on the same level as Domain person, since it's simply the ElasticSearch representation of what we need.